changelog/issue-244.md

```diff
@@ -0,0 +1,5 @@
+level: minor
+reference: issue 244
```

Comment:
https://github.com/taskcluster/taskcluster/pull/244 doesn't look relevant.. I think there's not a bug for this in terms of Taskcluster, so can just omit the reference.